backend/.env.example

```
@@ -1,33 +1,6 @@
-SUMUP_API_KEY="" # see doc/sumup.md, sup_sk_LZFWoLyd...
-SUMUP_APPLICATION_ID=
-SUMUP_MERCHANT_CODE=""
-SUMUP_PUBLIC_URL="" # needs to be an https URL
-VAT_RATES_0=0:Exempt
-VAT_RATES_1=7:Reduced
-VAT_RATES_2=19:Standard
+APP_CORS_ALLOW_ORIGINS=https://localhost:3000,http://localhost:8080
+APP_ENV=development
+APP_FRONTEND_URL=http://localhost:3000/
+APP_GIN_MODE=debug
+APP_REDIS_URL="redis://127.0.0.1:6379/0"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)